src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -39,12 +45,18 @@
         try (Session session = HibernateUtil.getSessionFactory().openSession()) {
             return Optional.ofNullable(session.get(Movie.class, id));
         } catch (Exception e) {
-            throw new DataProcessingException("Can't get a movie by id: " + id, e);
+            throw new DataProcessingException(CAN_T_GET_A_MOVIE_BY_ID_MSG + id, e);
         }
     }
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> fromMovie =
+                    session.createQuery("FROM Movie ", Movie.class);
+            return fromMovie.getResultList();
+        } catch (Exception e) {
+            throw new DataProcessingException(CAN_T_GET_ALL_MOVIES_MSG, e);
```

Comment:
Consider catching specific exceptions like `HibernateException` instead of a generic `Exception` to improve error handling and avoid catching unrelated exceptions.